File: app/src/main/java/com/ps/simplepapersoccer/GameObjects/Player.java
package com.ps.simplepapersoccer.GameObjects;

public class Player {
	public String playerName;
	public int playerNumber;
	public int playerColor;
	public Node goalNode;
	public int score;
	public boolean isAi;
	
	public Player(String name, int number, int color, boolean isAi)
	{
		this.playerName = name;
		this.playerNumber = number;
		this.playerColor = color;
		this.isAi = isAi;
	}
}

File: app/src/main/java/com/ps/simplepapersoccer/GameObjects/Player.kt
package com.ps.simplepapersoccer.GameObjects

import com.ps.simplepapersoccer.GameObjects.Game.Node

class Player(var playerName: String, var playerNumber: Int, var playerColor: Int, var isAi: Boolean) {
    var goalNode: Node? = null
    var score: Int = 0
}
